test/e2e/test_helpers.go

```diff
@@ -580,6 +581,35 @@ func (t testHelper) verifyCategoriesNutanixMachines(ctx context.Context, cluster
 	}
 }
 
+type verifyMemorySizeOnNutanixMachinesParams struct {
+	clusterName            string
+	namespace              *corev1.Namespace
+	toMachineMemorySizeGib int64
```

Comment:
this should ideally be a `resource.Quantity` type

Reply:
we should change it here as well then https://github.com/nutanix-cloud-native/prism-go-client/blob/main/v3/v3_structs.go#L255

Reply:
keeping it as is as later it has to be converted